### DISCUSSION TITLE
Preparing Files for Simulation

### DISCUSSION THREAD

**Discussion by Croker51o** [simnibs/simnibs#438] [7.Creating head models] [answered]
Dear Experts,

I’m new to tDCS simulations and have just started learning. I’m struggling with creating mesh files, and I would really appreciate your help. Could anyone explain the process of generating a mesh file from MRI data, starting from converting DICOM files to NIfTI files and then creating the mesh? I’d be grateful for any advice or insights from those with more experience in this area.

Best regards,
Cr

**Comment by jduemose** [comment, marked as answer]
Hi

SimNIBS does not handle DICOM to NIfTI conversion but you can use a tool like [dcm2niix](https://github.com/rordenlab/dcm2niix) or whatever you prefer. 

Once this is done, you will need to run the tool called CHARM which creates the physical head model. See  [this](https://simnibs.github.io/simnibs/build/html/tutorial/head_meshing.html) tutorial. It is just one command so quite straight-forward, however, it is important to check the quality of the results!

Finally, running a tDCS simulation can be done either in the GUI or by scripting (matlab or python). The former is shown [here](https://simnibs.github.io/simnibs/build/html/tutorial/gui.html) whereas you can find information on the latter approach [here](https://simnibs.github.io/simnibs/build/html/tutorial/scripting.html).

Hope this will get you started. Let us know if you have problems.

Best,
Jesper

**Reply by Croker51o** [reply]
Hi
Thank you very much for your valuable feedback. I would like to refer to it as I work on creating the files. If I have any questions, may I ask you again?

**Reply by jduemose** [reply]
yes, of course. Let us know if you experience any problems
